Magento folder content tune up (#815)
magento/merchdocs: src/magento/magento-extensions.md

## Inventory Management

> Magento [Inventory Management]({% link catalog/inventory-management.md %}) provides enhanced stock and shipment management  across one or multiple locations and sales channels with concurrent checkout protection and shipment matching algorithms. Track your inventory quantities, provide accurate salable stock amounts to customers, and ship according to recommendations or manual selections to control your entire inventory. Configure management settings globally, per source, and per product.

Reviewer: ```suggestion
Magento [Inventory Management]({% link catalog/inventory-management.md %}) provides enhanced stock and shipment management across one or multiple locations and sales channels with concurrent checkout protection and shipment matching algorithms. Track your inventory quantities, provide accurate salable stock amounts to customers, and ship according to recommendations or manual selections to control your entire inventory. Configure management settings globally, per source, and per product.
```

For technical, developer, and extension development information, see our developer documentation:

- [Inventory Architecture](https://devdocs.magento.com/guides/v2.3/inventory/index.html)
- [Order Processing with Inventory Management](https://devdocs.magento.com/guides/v2.3/rest/tutorials/inventory/index.html) tutorial
- REST API content, starting with [Managing sources](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/manage-sources.html)

> Inventory Management installs with all features enabled by default. No additional steps are required for enabling these inventory features.

Reviewer: ```suggestion
Inventory Management installs with all features enabled by default. No additional steps are required to enable these inventory features.
```

{:.bs-callout-info}
These features and extensions were developed as part of the [Magento Inventory](https://github.com/magento/inventory) (formerly MSI) project through the Magento Community Engineering program.

## Google reCAPTCHA

> Google reCAPTCHA provides a greater level of security for both the storefront and Admin UI than is available with standard CAPTCHA, and gives you the ability to:

Reviewer: ```suggestion
Google reCAPTCHA provides a greater level of security for both the storefront and Admin UI than is available with standard CAPTCHA and gives you the ability to:
```

- Verify when customers create accounts, retrieve passwords, log in to their accounts, or contact your company.
- Enhance security when Admin users log in.

It reduces potential user error when entering a Captcha code and encourages cart conversion without adding hurdles during checkout. [Enable and configure reCAPTCHA]({% link stores/security-google-recaptcha.md %}) using invisible or interactive checks to enhance secure access to the Magento Admin and storefront.

## Two-Factor Authentication

> The Magento Admin provides all access to your store, orders, and customer data. Magento Two-Factor Authentication (2FA or TFA) improves security by requiring [two-step authentication]({% link stores/security-two-factor-authentication.md %}) to access the Magento Admin from all devices. The extension supports multiple authenticators including Google Authenticator, Authy, Duo, and U2F keys. This applies to Magento Admin users only. It is not available for storefront customer accounts.

Reviewer: in other topics, it is Two-factor Authentication.   uncap Factor?

Author: I think we are considering it a proper name here.

> The Magento Admin provides all access to your store, orders, and customer data. Magento Two-Factor Authentication (2FA or TFA) improves security by requiring [two-step authentication]({% link stores/security-two-factor-authentication.md %}) to access the Magento Admin from all devices. The extension supports multiple authenticators including Google Authenticator, Authy, Duo, and U2F keys. This applies to Magento Admin users only. It is not available for storefront customer accounts.

Reviewer: ```suggestion
The Magento Admin provides all access to your store, orders, and customer data. Magento Two-Factor Authentication (2FA or TFA) improves security by requiring [two-factor authentication]({% link stores/security-two-factor-authentication.md %}) to access the Magento Admin from all devices. The extension supports multiple authenticators including Google Authenticator, Authy, Duo, and U2F keys. This applies to Magento Admin users only. It is not available for storefront customer accounts.
```
